lib/board.rb

```diff
@@ -0,0 +1,50 @@
+
+class Board
+  attr_accessor :cells
+
+
+  def initialize
+    @cells = [" "] * 9
```

Comment:
Why not just call the `reset!` method you define below here? It's always better to rely on an abstraction like a method call than literal logic.